static_report/src/pages/CRColumnDetailsPage.tsx

```diff
@@ -18,36 +18,39 @@ import {
 } from '../utils/transformers';
 
 import type { ComparisonReportSchema } from '../types';
+import { BreadcrumbNav } from '../components/shared/BreadcrumbNav';
+import { COLUMN_DETAILS_ROUTE_PATH } from '../utils/routes';
+import { NoData } from '../components/shared/NoData';
 interface Props {
   data: ComparisonReportSchema;
+  columnName: string;
```

Comment:
I found in the `types/Selectable` using the same properties, does it can extract a type?

Reply:
I think since this is on page-level, we should leave outside of the component library work. 
and it's slightly different here, as these route param props are required, not optional. 
```
export type Selectable = {
  onSelect: ({
    tableName,
    columnName,
  }: {
    tableName?: string;
    columnName?: string;
  }) => void;
};
```